src/reducers/tabs.ts

```diff
@@ -0,0 +1,111 @@
+import {combineReducers} from 'redux';
```

Comment:
you might want to make a structure like this
```
src/reducers/tabs.ts
   index.ts (this file)
   index.test.ts
   result.ts
   result.test.ts
   shelf.ts
   shelf.test.ts
   title.ts
   title..test.ts
```

So the reducer folder structure match state shapes

Reply:
Same for action folder and perhaps model?

Reply:
Maybe this is fine to do in a follow up PR though

Reply:
Seeing how you mix Tab(management)Action with SingleViewTab, I wonder if a better folder structure is 

```
src/reducers/tabs.ts
src/reducers/plot-screen   (or "/single-view-screen" depend on how we call it)
   result.ts
   result.test.ts
   shelf.ts
   shelf.test.ts
   title.ts
   title..test.ts
```

Reply:
I can understand this folder structure, except that I think it’s a bit inconsistent in two ways:
(1) we are using “tab” and “screen” to refer to the same thing -- so I'm leaning towards `tab` and `single-view-tab`.
(2) in the store state, `result`, `title`, and `shelf` are nested under `tabs`, but in the folder structure it is not nested -- should we nest the folder structure in the same way we nest the state or is that too confusing?

Reply:
Discussed on slack, let's keep it flat!